tests/virt_autotest/host_upgrade_step2_run.pm

```diff
@@ -14,20 +14,31 @@ use base "host_upgrade_base";
 #use virt_utils qw(set_serialdev);
 use testapi;
 use strict;
+use virt_utils;
 
 sub get_script_run {
     my $self = shift;
 
     my $pre_test_cmd = $self->get_test_name_prefix;
     $pre_test_cmd .= "-run 02";
 
-    return "$pre_test_cmd";
+    return "rm /var/log/qa/old* /var/log/qa/ctcs2/* -r;" . "$pre_test_cmd";
 }
 
 sub run {
     my $self = shift;
     $self->run_test(12600, "Host upgrade to .* is done. Need to reboot system|Executing host upgrade to .* offline",
         "no", "yes", "/var/log/qa/", "host-upgrade-prepAndUpgrade");
+
+    #replace module url with openqa daily build modules link
+    my $upgrade_product = get_required_var('UPGRADE_PRODUCT');
+    $upgrade_product = lc($upgrade_product);
+    $upgrade_product =~ /sles-([0-9]+)-sp/;
+    my $upgrade_release = $1;
```

Comment:
how about a single line `my ($upgrade_release) = lc($upgrade_product) =~ /sles-([0-9]+)-sp/;`

Reply:
Good idea. Change to it.